woopra/src/main/java/woopra/Constants.java

```diff
@@ -1,4 +1,4 @@
-package com.woopra.tracking.android;
+package woopra;
 
```

Comment:
imo: Should we change the package from `com.woopra.tracking.android` to `woopra`? It seems we should not modify the package name.